server/server.go

```diff
@@ -461,6 +461,16 @@ func getInitSetting(ctx context.Context, store *store.Store) (*config, error) {
 		return nil, err
 	}
 
+	// initial feishu app
+	if _, err := store.CreateSettingIfNotExist(ctx, &api.SettingCreate{
```

Comment:
nit: I see why we did this before. I think we need an upsert() interface, but let's worry about it later.